### ISSUE TITLE
class attribute cannot be passed to shopp('product.variations')

### ISSUE THREAD

**Issue by meredevelopment** [ingenesis/shopp#3225]
There is no handling for a 'class' attribute in the shopp('product.variations') function. Ideally a class should be added to the <select> tag when called like this:

```
<?php shopp( 'product.variations', 'mode=multiple&class=myclass'); ?>
```

This is pretty important when using a grid/framework like Foundation or Bootstrap.

**Comment by msigley** [comment]
product.variations has before_menu and after_menu options. maybe that is why it doesn't have a class option?

product.variations is also one of these weird calls with multiple uses. we really should separate the loop or the menu part out into its own function, which would make this option easier to implement. Something like product.variations-menu.

**Comment by meredevelopment** [comment]
Yes I see those options, which can be useful, but in the case where a class needs to be passed to the <select> tag/s it's not going to help. On a dev site here I've added the 'class' option myself, in almost exactly the same was as in my PR for adding classes to the orderby_list function:

https://github.com/meredevelopment/shopp/commit/d4517863e9dcb3707fd9dc2d847b8f1563f68054

Reckon I should do a PR for product.variations too?
